packages/sdk/src/index.ts

```diff
@@ -222,6 +222,11 @@ export interface ConsolidatedGraphemeGuardReport extends ConsolidatedReport {
    * A user-friendly description of the grapheme type.
    * */
   grapheme_description: string;
+
+  /**
+   * The canonical status for the current grapheme.
+   * */
+  is_canonical: boolean;
```

Comment:
Is_canoncial is not in ConsolidatedGraphemeGuardReport, we need new type: ConfusableGuardReport extends ConsolidatedGraphemeGuardReport